tools/tests/test_lint.py

```
@@ -67,6 +86,111 @@ def test_clean_tree_passes(tmp_path):
 
 # --- zone wall -------------------------------------------------------------
 
+def test_delivery_fires_when_the_charter_is_missing(tmp_path):
+    make_clean_tree(tmp_path)
+    (tmp_path / "charter" / "CHARTER.md").unlink()
+    findings = lint.run(tmp_path)
+    assert len(findings) == 1 and "delivery" in findings[0]
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)